types/src/block_info.rs

```diff
@@ -157,6 +155,24 @@ impl BlockInfo {
     pub fn version(&self) -> Version {
         self.version
     }
+
+    /// This function checks if the current BlockInfo has
+    /// exactly the same values in those fields that will not change
+    /// after execution, compred to a given BlockInfo
+    pub fn match_info(&self, block_info: &BlockInfo) -> bool {
```

Comment:
also better to rename to match_ordered_only

Reply:
resolved